doc-site/docusaurus.config.js

```diff
@@ -0,0 +1,112 @@
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+const child_process = require('child_process');
+
+const remoteCommonCssFile = 'https://raw.githubusercontent.com/commitdev/zero/doc-site/doc-site/src/css/custom.css';
+const cssDownloadPath = './src/css/zero-downloaded-global-custom.css';
+
+function downloadCss(url, saveTo) {
+  child_process.execFileSync('curl', ['--silent', '-f', '-L', url, '-o', saveTo], {encoding: 'utf8'});
+  return require.resolve(saveTo);
+}
+
+// This is to download the CSS from commitdev/zero instead of using the local copy, this
+// allows us to maintain only one set of global css throughout the modules and have
+// consistent look and feel throughout the modules, this is synchronous but only runs during build time
+let customCss;
+try {
+  customCss = downloadCss(remoteCommonCssFile, cssDownloadPath)
+} catch (e) {
+  customCss = require.resolve('./src/css/fallback.css');
+}
```

Comment:
The weird thing to note here is I'm downloading the global css file from `zero` repo then if it fails, it uses its original copy instead. Its a synchronous action but it only happens during build time in CI.